src/ripple/app/paths/PathRequest.h

```diff
@@ -112,11 +118,12 @@ class PathRequest :
     boost::optional<AccountID> raSrcAccount;
     boost::optional<AccountID> raDstAccount;
     STAmount saDstAmount;
+    boost::optional<STAmount> saSendMax;
 
     std::set<Issue> sciSourceCurrencies;
     std::map<Issue, STPathSet> mContext;
 
-    bool bValid;
+    bool convert_all_;
```

Comment:
This is cryptic!  :-)  A comment here would be helpful.

And also should be renamed to look like the other member variables, sorry.